src/Products/ZCatalog/ZCatalog.py

```diff
@@ -410,14 +410,14 @@ def manage_clearIndex(self, ids=None,
 
     @security.protected(manage_zcatalog_entries)
     def reindexIndex(self, name, REQUEST, pghandler=None):
-        if isinstance(name, str):
-            name = (name, )
-
+        # This method does the actual reindexing of indexes.
+        # `name` can be the name of an index of a list of names.
```

Comment:
Thank you @icemac ! This makes the codebase more accessible!

P.S.:
One more typo:  `of` -> `or`

Reply:
Fixed on master.